### PR TITLE
Refactor register page and general cleanup

### PR DESCRIPTION
**Pull request by puckzxz**
- Adjust our register backend route to return a `409` if a user already exists
- General frontend cleanup
- Refactors the register page to include meaningful messages when trying to create an account

### CODE REVIEW

File: backend/routes/auth.go [hunk 1]
```diff
@@ -85,7 +85,7 @@ func (r *Routes) Register(resp http.ResponseWriter, req *http.Request) {
 	}
 
 	if exists {
-		resp.WriteHeader(http.StatusBadRequest)
+		resp.WriteHeader(http.StatusConflict)
```
Comment (RaulAkerman) [inline backend/routes/auth.go:88]: 👍🏻

**Review by ImVexed — APPROVED** [review]

File: frontend/src/app/clips/[id]/page.tsx [hunk 2]
```diff
@@ -1,53 +1,12 @@
...
-  //   console.log(player)
-  //   loadAsset();
-  // }, [controllerRef, params.id]);
-  // console.log(videoRef);
-  // console.log(params.id);
-
   return (
     <main className="bg-white dark:bg-black max-w-[70%] mt-6 mx-auto">
-      <ShakaPlayer src={`http://localhost:8080/api/clips/${params.id}/dash.mpd`} ref={videoRef} autoPlay={true} chromeless={false}/>
+      <ShakaPlayer src={`http://localhost:8080/api/clips/${params.id}/dash.mpd`} autoPlay={true} />
```
Comment (RaulAkerman) [inline frontend/src/app/clips/[id]/page.tsx:9]: 👍🏻

File: frontend/src/app/page.tsx [hunk 3]
```diff
@@ -29,17 +24,20 @@ export default function Home() {
...
                   <figure className={video.processing ? "flex-auto grow self-center pt-8 glass w-full h-full" : ""}>
-                    {video.processing
-                      ? <div>Processing!</div>
-                      : <img src={`http://localhost:8080/api/clips/${video.id}/thumbnail.jpg`} />
-                    }
+                    {video.processing ? (
+                      <div>Processing!</div>
+                    ) : (
+                      <img src={`http://localhost:8080/api/clips/${video.id}/thumbnail.jpg`} />
+                    )}
```
Comment (RaulAkerman) [inline frontend/src/app/page.tsx:31]: 💯

**Review by RaulAkerman — APPROVED** [review]
Looks good! 💯

**Pull request merged**